lib/ansible/cli/pull.py

```diff
@@ -37,7 +37,8 @@ class PullCLI(CLI):
         each set to run via cron and update playbook source via a source repository.
         This inverts the default *push* architecture of ansible into a *pull* architecture,
         which has near-limitless scaling potential.
-
+        ``ansible-pull`` is not meant to run in parallel on the same host against the same checkout. You should either
+        add a lock (``setlock/flock``) around the invocation or use separate directories.
```

Comment:
```suggestion
        To make concurrent ``ansible-pull`` invocations for the same checkout on the same host, you should 
        use either locks (``setlock/flock``) or separate directories.
```

Reply:
@saranti Thanks for the contribution! I've a suggestion to address the reader more directly. Hopefully it doesn't alter your original meaning.